crates/solidity/outputs/npm/tests/src/parser/parse.test.mts

```diff
@@ -0,0 +1,60 @@
+import { Parser } from "@nomicfoundation/slang/parser";
+import { NonterminalKind, TerminalKind, assertNonterminalNode, assertTerminalNode } from "@nomicfoundation/slang/cst";
+import { LanguageFacts } from "@nomicfoundation/slang/utils";
+
+function new_parser(): Parser {
+  return Parser.create(LanguageFacts.latestVersion());
+}
+
+test("parse terminal", () => {
+  const source = "about_time";
+  const parser = new_parser();
+
+  const tree = parser.parseNonterminal(NonterminalKind.ImportDeconstructionSymbol, source).tree;
```

Comment:
is this intended? `parseNonterminal()`?

Reply:
Also `parseFileContents()` in `parse nonterminal` below..

Reply:
note that we don't have a way to directly parse a terminal (maybe we should?)